eo-maven-plugin/src/test/java/org/eolang/maven/VerifyMojoTest.java

```diff
@@ -63,7 +64,17 @@ void doesNotFailWithNoErrorsAndWarnings(@TempDir final Path temp) {
         );
     }
 
+    /**
+     * The test is disabled, needs to enable it and remove this commentary.
+     * @param temp Temp.
+     * @param out Out.
+     * @todo #2890:30min Fix this flaky test and enable it. It failed in ci
```

Comment:
@levBagryansky it would be better to place the todo above the class and make a link to the method via `{@link}`

Reply:
@maxonfjvipon done